pkg/knode-manager/utils/expansion/expand.go

```diff
@@ -0,0 +1,69 @@
+package expansion
+
```

Comment:
what does expansion use for

Reply:
Parsing variable templates, like $(varx)